Ports/libiconv/patches/0001-Stub-out-getprogname-for-serenity.patch

```diff
@@ -8,13 +8,13 @@ Subject: [PATCH 1/2] Stub out getprogname() for serenity
  1 file changed, 2 insertions(+)
 
 diff --git a/srclib/getprogname.c b/srclib/getprogname.c
-index 22d71eb..7cc112d 100644
+index 62a4800..40ee7ec 100644
 --- a/srclib/getprogname.c
 +++ b/srclib/getprogname.c
-@@ -244,6 +244,8 @@ getprogname (void)
-         }
+@@ -287,6 +287,8 @@ getprogname (void)
+       close (fd);
      }
-   return NULL;
+   return "?";
 +# elif defined __serenity__
 +  return "?";
```

Comment:
This patch can probably be removed entirely, `getprogname` has been implemented some time after `make` has been initially ported.

Reply:
Yes, you are right.